examples/transforms/transform.rs

```diff
@@ -0,0 +1,148 @@
+use bevy::prelude::*;
+
+use std::f32::consts::PI;
+
+// A struct for additional data of for a moving cube.
+struct CubeState {
+    start_pos: Vec3,
+    move_speed: f32,
+    turn_speed: f32,
+}
+
+// A struct adding information to a scalable entity,
+// that will be stationary at the center of the scene.
+struct Center {
+    max_size: f32,
+    min_size: f32,
+    scale_factor: f32,
+}
+
+fn main() {
+    App::build()
+        .add_plugins(DefaultPlugins)
+        .add_startup_system(setup.system())
+        .add_system(move_cube.system())
+        .add_system(rotate_cube.system())
+        .add_system(scale_down_sphere_proportional_to_cube_travel_distance.system())
+        .run();
+}
+
+// Startup system to setup the scene and spawn all relevant entities.
+fn setup(
+    mut commands: Commands,
+    mut meshes: ResMut<Assets<Mesh>>,
+    mut materials: ResMut<Assets<StandardMaterial>>,
+) {
+    // Add an object (sphere) for visualizing scaling.
+    let center_sphere_mesh = PbrBundle {
+        mesh: meshes.add(Mesh::from(shape::Icosphere {
+            radius: 3.0,
+            subdivisions: 32,
+        })),
+        material: materials.add(Color::YELLOW.into()),
+        transform: Transform::from_translation(Vec3::ZERO),
+        ..Default::default()
+    };
+    commands.spawn_bundle(center_sphere_mesh).insert(Center {
+        max_size: 1.0,
+        min_size: 0.1,
+        scale_factor: 0.05,
+    });
```

Comment:
The extra variable is unnecessary, as it isn't used anywhere else.
```suggestion
    // Add an object (sphere) for visualizing scaling.
    commands
        .spawn_bundle(PbrBundle {
            mesh: meshes.add(Mesh::from(shape::Icosphere {
                radius: 3.0,
                subdivisions: 32,
            })),
            material: materials.add(Color::YELLOW.into()),
            transform: Transform::from_translation(Vec3::ZERO),
            ..Default::default()
        })
        .insert(Center {
            max_size: 1.0,
            min_size: 0.1,
            scale_factor: 0.05,
        });
```

Reply:
I've nothing in particular against your suggestion, but I actually purposefully moved the bundle creation out of the statement to have more but less convoluted statements. What would be preferred for examples?

Reply:
Currently, practically all examples construct the bundles inside the command calls.
The exceptions do this if a particular value is more difficult to determine.